### PR TITLE
[mellanox]: Move FW upgrade script to base image.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

File: files/build_templates/swss.service.j2 [hunk 1]
```diff
@@ -18,8 +18,10 @@ ExecStartPre=/usr/bin/docker exec database redis-cli -n 1 FLUSHDB
 ExecStartPre=/usr/bin/docker exec database redis-cli -n 2 FLUSHDB
 
 {% if sonic_asic_platform == 'mellanox' %}
-ExecStartPre=/etc/init.d/sxdkernel start
+TimeoutStartSec=3min
 ExecStartPre=/usr/bin/mst start
+ExecStartPre=/usr/bin/mlnx-fw-upgrade.sh
```
Comment (qiluo-msft) [inline files/build_templates/swss.service.j2]: If firmware is really upgraded, will the service restart?

**Comment by qiluo-msft** [comment]
> 	exit 0

The comment is not updated.
Could you not use the 'exit 0' trick? Use a normal non-zero return value?

---
Refers to: platform/mellanox/mlnx-fw-upgrade.sh:52 in ed93a79. [](commit_id = ed93a795eae0eade4a9035165d40d2c04b865531, deletion_comment = False)

File: files/build_templates/swss.service.j2 (hunk 1 above)
Comment (oleksandrivantsiv) [inline files/build_templates/swss.service.j2]: No need to restart service. FW update procedure starts before SX kernel drivers and SDK.

**Comment by oleksandrivantsiv** [comment]
@qiluo-msft I've updated mlnx-fw-upgrade.sh script. Please review

**Comment by marian-pritsak** [comment]
retest this please

**Pull request merged**